src/index.js

```diff
@@ -31,13 +38,33 @@ tasks.forEach((element) => {
   div.appendChild(div2);
   const checkbox = document.createElement('input');
   checkbox.type = 'checkbox';
+  checkbox.className = 'check';
   div2.appendChild(checkbox);
   const para = document.createElement('p');
   para.innerText = element.description;
   div2.appendChild(para);
+  checkbox.addEventListener('click', () => {
```

Comment:
- [ ] You used a click listener but it is required for this project to use an on-change listener, please switch to that one in order to comply with the project requirements.

Reply:
Okay, I've applied this.